tests/shanghai/eip4895_withdrawals/test_withdrawals.py

```diff
@@ -373,7 +373,7 @@ def test_self_destructing_account(
         sender=sender,
         gas_limit=100000,
         to=self_destruct_contract_address,
-        data=left_pad_zeros_up_to_size(recipient, 32),
+        data=Hash(recipient, left_padding=True),
```

Comment:
lets name Hash Hash32 because in tests we have Hash8 (block nonce) and Hash20 (address)